### PR TITLE
Feature contact form: mobile portfolio project

### PR DESCRIPTION
**Pull request by jahongiry**
**Contact form added to the mobile project**
-Html
-Ccs (flex box)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -315,5 +315,58 @@ <h5 class="about-me-heading">About Me</h5>
...
+        ></a>
+        <a href="#"
+          ><ion-icon class="icons-last" name="logo-linkedin"></ion-icon
+        ></a>
+        <a href="#"
+          ><ion-icon class="icons-last" name="logo-twitter"></ion-icon
+        ></a>
+        <a href="#"
+          ><ion-icon class="icons-last" name="logo-whatsapp"></ion-icon
+        ></a>
```
Comment (CollinsTatang) [inline index.html:368]: - [ ] Kindly wrap your social icons with `ul` and each in a `li` with a link `a tag`.

File: index.html [hunk 2]
```diff
@@ -315,5 +315,58 @@ <h5 class="about-me-heading">About Me</h5>
...
+              maxlength="500"
+              placeholder="Enter your text here"
+              required
+            >
+            </textarea>
+          </li>
+          <li>
+            <button type="button" class="get-in-touch">Get in touch</button>
+          </li>
+        </ul>
```
Comment (CollinsTatang) [inline index.html:353]: - [ ] Please remove the `ul tag` and the `li` from the form `input` for a good HTML semantic practice

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @jahongiry and @SirriCelles , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required Changes ♻️

- [ ] Great job implementing the project design as per the project requirement design but your form is not responsive to the different mobile view (screen size).
### **Take  A Look**
responsive



_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gtekle — CHANGES_REQUESTED** [review]
Hi **@TEAM**, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
dog-pug


 ## Highlights
:heavy_check_mark: Clean code
:heavy_check_mark: Good use of Github flow

 ## Required Changes ♻️
_Check the comments under the review._

- [ ] As mentioned by the previous reviewer, you should update the README to include the contributors, replace the `**Author1**` Placeholder text with the real name, the getting started section should include steps to clone and run the project locally, etc. **NB**: professional readme is as important as a clean and working code.
readme-impl


- [ ] You should remove the fixed width for the sections for it is making your website not responsive. Take a look at the following snapshot.
mobile-responsiveness-impl

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AnselemOdims — APPROVED** [review]
### Hi  🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**